packages/expo-router/jest.config.js

```diff
@@ -12,15 +13,26 @@ function withDefaults({ watchPlugins, ...config }) {
     roots: ['src'],
     clearMocks: true,
     setupFilesAfterEnv: ['./build/testing-library/mocks.js'],
+    testPathIgnorePatterns: ['__typetests__/*'],
   };
 }
 
-module.exports = withWatchPlugins({
```

Comment:
@EvanBacon I'm getting this error when I'm trying to use `withWatchPlugins` and the `tsd` test runner

```
 FAIL  src/__typetests__/typed-routes.test.ts
  ● Test suite failed to run

    prompt() {
              return `select which PLATFORMS to run ${chalk.italic(this._getActiveProjectsText())}`;
            } could not be cloned.

      at ChildProcessWorker.send (../../node_modules/jest-worker/build/workers/ChildProcessWorker.js:402:17)
```

Tests pass when I run as a single project. Just fail when I run all projects. Is there another config option I need on the jest project?

Reply:
I think watch plugins need to be applied to the individual projects or the entire project, but not both.